ingest/muxer.go

```diff
@@ -1831,6 +1836,24 @@ loop:
 			}
 			continue
 		}
+		// Make sure the version is new enough
+		if ig.ew.serverVersion < im.minVersion {
```

Comment:
I think this needs an additional check.  

If minVersion > 0 && serverVersion < minVersion

Reply:
yes I am a moron and yes I realized it several hours later.